server/jetstream_api.go

```diff
@@ -862,10 +869,25 @@ func (js *jetStream) apiDispatch(sub *subscription, c *client, acc *Account, sub
 	// Copy the state. Note the JSAPI only uses the hdr index to piece apart the
 	// header from the msg body. No other references are needed.
 	// Check pending and warn if getting backed up.
-	const warnThresh = 128
+	s.optsMu.RLock()
+	limit := s.opts.JetStreamRequestQueueLimit
```

Comment:
For now make a global var that is set to constant but we can override on config parse?